README.md

```diff
@@ -27,8 +27,7 @@ this is not for you. For such devices, see
     VMs
   * My dumped VBIOS did **not** have EFI support, but I don't think that is
     necessary when the card has no outputs. rom-parser output for reference:
-    ```
-Valid ROM signature found @0h, PCIR offset 190h
+    ```Valid ROM signature found @0h, PCIR offset 190h
```

Comment:
This produces the wrong output. Appending a line to the triple-backticks sets a syntax-highlighting and won't show the text.

Reply:
Currently it looks like this:
![image](https://user-images.githubusercontent.com/7986123/34339527-95928744-e975-11e7-8ef8-5c064294f8cf.png)
And with the back ticks on the same row:
![image](https://user-images.githubusercontent.com/7986123/34339539-c6473524-e975-11e7-9267-0e88d4d047c4.png)

Reply:
Correct. Now compare both: Your amendments swallow the first line of the output

Reply:
Funny :v: